thentos-purescript/src/Mula.js

```diff
@@ -0,0 +1,59 @@
+/* global exports */
+"use strict";
+
+// module Mula
+exports.tr = function(key) {
+    return table[lang][key];
+};
+
+var lang = "en";
+
+var table = {};
+
+table.en = {
+    "TR__CANCEL": "cancel",
+    "TR__EMAIL": "email",
+    "TR__ERROR_FORMAT_EMAIL": "Email format is incorrect.",
+    "TR__ERROR_MATCH_PASSWORD": "Passwords do not match.",
+    "TR__ERROR_REQUIRED_EMAIL": "Email is required.",
+    "TR__ERROR_REQUIRED_PASSWORD": "Password is required.",
+    "TR__ERROR_REQUIRED_TERMS_AND_CONDITIONS": "You must agree to terms and conditions",
+    "TR__ERROR_REQUIRED_CAPTCHA_SOLUTION": "Please enter the keyword displayed above",
+    "TR__ERROR_REQUIRED_USERNAME": "Username is required",
+    "TR__ERROR_TOO_SHORT_PASSWORD": "Password is too short.",
+    "TR__I_ACCEPT_THE_TERMS_AND_CONDITIONS": "I accept the [[link:terms and conditions]].",
+    "TR__PASSWORD": "password",
+    "TR__PASSWORD_REPEAT": "password repeat",
+    "TR__REGISTER": "register",
+    "TR__REGISTER_SUCCESS": "Thank you for your registration. We sent you an email with an activation link. In case you do not recieve that email, please also check your spam folder.",
+    "TR__REGISTRATION_CALL_FOR_ACTIVATION": "Once you have clicked on the activation link you can proceed using {{siteName}}.",
```

Comment:
Some of the translation lines are too long (> 100 chars), but I guess that can be tolerated.